htk-jenkins/entrypoint

```diff
@@ -5,11 +5,11 @@ if [ ! -s /var/jenkins_home/config.xml ];then
 	cp -r /var/jenkins_home_default/./ /var/jenkins_home/
 fi
 export JAVA_OPTS+=" -Dorg.csanchez.jenkins.plugins.kubernetes.pipeline.ContainerExecDecorator.websocketConnectionTimeout=180"
-export JAVA_OPTS+=" -Xloggc:$JENKINS_HOME/gc-%t.log -XX:NumberOfGCLogFiles=5 -XX:+UseGCLogFileRotation -XX:GCLogFileSize=20m -XX:+PrintGC -XX:+PrintGCDateStamps -XX:+PrintGCDetails -XX:+PrintHeapAtGC -XX:+PrintGCCause -XX:+PrintTenuringDistribution -XX:+PrintReferenceGC -XX:+PrintAdaptiveSizePolicy"
+export JAVA_OPTS+=" -Xlog:gc:$JENKINS_HOME/gc-%t.log -Xlog:gc* -Xlog:gc+heap=trace -Xlog:age*=debug -Xlog:ref*=trace -Xlog:ergo*=trace"
```

Comment:
these options look correct as far as I can judge, however, are we really need trace?

also, other options can be addressed similarly:
```
-XX:NumberOfGCLogFiles=5 -XX:GCLogFileSize=20m
```
I used to do something like below:
```
-Xlog:gc*,safepoint:/tmp/hive/gc-%t.log:time,uptime:filecount=5,filesize=20M
```

Reply:
I haven't used these logs in the context of Jenkins so I just tried to replicate what the previous options used to do. I am OK if we want to reduce the verbosity.

Reply:
iirc there were some memory issues in the beginning; these were most likely added around that time.

the most important is to give a bit more mem to jenkins than the default - as it needs to be prepared to control around a ~100 docker pods at a time

Reply:
I am not very opinionated about this. Let me know which version you prefer and I will apply the changes and merge it.

Reply:
I don't have experience with those logs, I believe by default let's not go more verbose than DEBUG
we'll increase the severity in case of investigations :)
filecount/filesize: let it be inline with the old values

other than that, it's fine

Reply:
Are we ok with https://github.com/kgyrtkirk/hive-test-kube/pull/16/commits/2dcac1636b3ebd6d39eb911fb67ae48da7b1e7cb ?

Reply:
I assume we are good so will merge this in a few minutes.